string.c

```diff
@@ -2981,13 +2982,13 @@ str_uplus(VALUE str)
 /*
  * call-seq:
  *   -string -> frozen_string
+ *   dedup -> frozen_string
  *
  * Returns a frozen, possibly pre-existing copy of the string.
  *
  * The returned \String will be deduplicated as long as it does not have
  * any instance variables set on it and is not a String subclass.
  *
```

Comment:
If we are going to list both in call-seq, the documentation should give examples of when it is best to use each (`-@` for ilteral strings, `dedup` for other cases).